descriptors/ModuleDescriptor-template.json

```diff
@@ -549,7 +549,7 @@
     },
     {
       "id": "item-storage",
-      "version": "6.1 7.0"
+      "version": "7.6"
```

Comment:
This is changing the major range of versions that the module supports. Personally I think this should be reflected in the implementation version (in the pom.xml). 

Is that already a new major version? If not, please increment that.

Reply:
Should I increment major version (i.e. to 18.0.0-SNAPSHOT) or minor version?

Reply:
Major version has been incremented.

Reply:
@bohdan-suprun Apologies, for the confusion. Mod-circulation was already awaiting a new major version (17) and so this did not need incrementing, please reverse this change.